apps/src/templates/AchievementDialog.jsx

```diff
@@ -162,6 +202,19 @@ const AchievementDialog = Radium(React.createClass({
           {this.achievementRow(!tooManyBlocks, this.blocksUsedMessage(blockDelta, params))}
           {this.achievementRow(!tooManyHints, this.hintsMessage(tooManyHints))}
         </div>
+        <div style={styles.stageRewards}>
```

Comment:
Sure. How does `g.stageprogress` look?

It'd be nice to keep putting new features behind flags. I'll look into adding an "experiment group" feature to simplify the process of enabling/disabling them in bulk.